api/app/graphql/types/task_type.rb

```diff
@@ -1,8 +1,6 @@
 module Types
   class TaskType < Types::BaseObject
-    field :id, ID
-    field :name, String
-    field :complete, Boolean
+    auto_field :id, :name, :complete
```

Comment:
This is maybe not a huge win for this tiny type, but you can imagine it would be amazing for a type with 20+ fields.